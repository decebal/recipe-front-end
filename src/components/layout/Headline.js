import styled from 'styled-components';

const Headline = styled.h1`
  color: #767676;
  padding-bottom: 1rem;
  border-bottom: 2px solid rgba(0,0,0,0.1);
  
  background: #b4ddb4;
  background: -moz-linear-gradient(top, hsl(120,38%,79%) 0%, hsl(120,38%,65%) 17%, hsl(120,38%,51%) 33%, hsl(120,100%,27%) 67%, hsl(120,100%,17%) 83%, hsl(120,100%,7%) 100%);
  background: -webkit-linear-gradient(top, hsl(120,38%,79%) 0%,hsl(120,38%,65%) 17%,hsl(120,38%,51%) 33%,hsl(120,100%,27%) 67%,hsl(120,100%,17%) 83%,hsl(120,100%,7%) 100%);
  background: linear-gradient(to bottom, hsl(120,38%,79%) 0%,hsl(120,38%,65%) 17%,hsl(120,38%,51%) 33%,hsl(120,100%,27%) 67%,hsl(120,100%,17%) 83%,hsl(120,100%,7%) 100%);
  filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#b4ddb4', endColorstr='#002400',GradientType=0 );
  
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
  font-weight: 200;
  font-size: 3rem;
  text-align: center;
  vertical-align: middle;
`;

export default Headline;
