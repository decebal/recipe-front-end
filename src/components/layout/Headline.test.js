import React from 'react';
import renderer from 'react-test-renderer';
import Headline from './Headline';


test('Headline renders correctly', () => {
  const tree = renderer
    .create(<Headline />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
