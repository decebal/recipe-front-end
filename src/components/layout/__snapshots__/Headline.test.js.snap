// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headline renders correctly 1`] = `
<h1
  className="sc-bdVaJa qBDhK"
/>
`;
